open-hackathon-server/src/hackathon/hack/register_manager.py

```diff
@@ -141,6 +144,15 @@ def delete_registration(self, args):
                 self.db.delete_object(register)
                 hackathon = self.hackathon_manager.get_hackathon_by_id(register.hackathon_id)
                 self.__update_register_stat(hackathon)
+
+                user = self.user_manager.get_user_by_id(register.user_id)
+                if not user:
+                    return not_found("this registered user is not found!")
+                team = self.team_manager.get_team_by_user_and_hackathon(user, hackathon)
+                if not team:
+                    return not_found("team of this registered user is not found!")
```

Comment:
Not a good idea to return `not_found`. `not_found` is a kind of `error`. Ususally, we should return `ok` when the item to delete doesn't exists. e.g. duplicated delete